sdk/python/feast/feature_store.py

```diff
@@ -809,6 +809,26 @@ def tqdm_builder(length):
                 feature_view, self.project, start_date, end_date
             )
 
+    @log_exceptions_and_usage
+    def write_to_online_store(
```

Comment:
@vas28r13 thx for adding this capability, we use this capability so we can write inference data in a real-time flow. However, the `get_feature_view` does not use the `allow_cache` option of the registry. This causes a lookup to refresh the registry for every call to `write_to_online_store`. This blocks the new capability from being used real-time. Would it make sense to add a PR that fixes that? thx...

Reply:
would you mind making an issue so we can track this?

Reply:
done